src/projection/scvelo/script.py

```diff
@@ -0,0 +1,88 @@
+import scvelo
+import mudata
+import logging
+from contextlib import redirect_stdout
+from pathlib import Path
+import matplotlib as mpl
+from collections import defaultdict
+from sys import stdout
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+console_handler = logging.StreamHandler(stdout)
+logFormatter = logging.Formatter("%(asctime)s %(levelname)-8s %(message)s")
+console_handler.setFormatter(logFormatter)
+logger.addHandler(console_handler)
+
+## VIASH START
+def none_factory():
+  return None
+
+par = defaultdict(none_factory, {
+  'input': './resources_test/rna_velocity/velocyto_processed/cellranger_tiny.loom',
+  'output': './foo',
+  'log_transform': True,
+  'n_neighbors': 30
+})
+## VIASH END
+mpl.rcParams['savefig.dpi']=150
+
+def main():
```

Comment:
can you add a comment mentioning why everything is a main?

Reply:
Done!